data/resources/ui/content-send-photo-dialog.ui

```diff
@@ -42,7 +42,7 @@
                     <property name="valign">end</property>
                     <style>
                       <class name="circular"/>
-                      <class name="suggested-action"/>
+                      <class name="accent"/>
```

Comment:
I prefer the previous style as the button is more distinct.

Reply:
Do you want to use this style only for photo dialog or for both buttons?
![image](https://user-images.githubusercontent.com/63008755/200176830-49e74caf-a55f-4ac0-a0a8-9c7c3ef611c4.png)

Reply:
Also I got a strange bug with that style that I can't reproduce with `accent` style
And it needs changes to css for correct transition and changes to the icon because it's not centered correctly
And I think that it looks unusual